google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/application/AppEngineApplicationCreateDialog.java

```diff
@@ -74,6 +71,11 @@ public AppEngineApplicationCreateDialog(@NotNull Component parent, @NotNull Stri
     super(parent, false);
     this.gcpProjectId = gcpProjectId;
     this.userCredential = userCredential;
+
+    Font font = parent.getFont();
+    HTML_OPEN_TAG =
+        "<html><font face=\"" + font.getName() + "\" size=\\\"" + font.getSize() + "\\\">";
```

Comment:
just curious, why the triple `\`? It looks like it would result in `size=\"12\"`. Also I don't think font size need double quotes around the number.

Reply:
The single quotes around the font size mean something a different: https://www.w3schools.com/tags/att_font_size.asp